packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3906,14 +3906,6 @@ describe('plugin-meetings', () => {
             checkScreenShareAudioPublished(audioShareTrack);
           });
         });
-        it('creates instance and publishes with annotation info', async () => {
```

Comment:
Instead of this test we should now probably test that when we publish a track that has annotation info, the correct annotation info is sent with the floor request. See the existing test "requests screen share floor and publishes the screen share video track", you can do a new test in a similar way or expand that existing test to check annotation info.